spec/latest/json-ld-framing/index.html

```diff
@@ -301,7 +305,10 @@
       <a href="../json-ld-api/#compaction">Compacting</a>.</dd>
     <dt><code>@omitDefault</code></dt>
     <dd>Used in <a href="#framing">Framing</a> to override the
-      value of <a>omit default flag</a> within a specific frame.</dd>
+      value of <a>omit default flag</a> within a specific frame.
+      <div class="issue">Is this flag really useful? Easier to simply not have a default value,
+        if it shouldn't match.</div>
```

Comment:
I'm not opposed to changing this behavior around, especially if it is confusing. We should probably add a few examples to the test suite as use cases for this... just want to make sure it's easy for people to configure when defaults appear and when they do not with a reasonable level of granularity. This shouldn't hold up the PR, it's just a discussion/implementation experience thing to consider later.